servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ClientParentConnectionContext.java

```diff
@@ -447,4 +498,21 @@ public void eventConsumed() {
             }
         }
     }
+
+    private static final class StacklessCancellationException extends CancellationException {
+        private static final long serialVersionUID = 3235852873427231209L;
+
+        private StacklessCancellationException() { }
+
+        // Override fillInStackTrace() so we not populate the backtrace via a native call and so leak the
+        // Classloader.
+        @Override
+        public Throwable fillInStackTrace() {
+            return this;
+        }
+
+        static StacklessCancellationException newInstance(Class<?> clazz, String method) {
```

Comment:
do we need to create the new exception type, or can we just use `CancellationException` and `ThrowableUtils.unknownStackTrace`?

Reply:
if we don't override `fillInStackTrace()`, the `Throwable` ctor will invoke it and fill in the stacktrace which `ThrowableUtils.unknownStackTrace` will override later => useless expensive operation.